### DISCUSSION TITLE
Enable Warning thresholds for Grafana Alerts

### DISCUSSION THREAD

**Discussion by AlexeiZenin** [grafana/grafana#62634] [Alerting]
With Datadog for the same monitor query we can specify the following thresholds in our Cloudformation. 

With moving to Grafana Alerts we found we need to duplicate our CF resource; 1 for the alert and 1 for a warning. The only differences between the 2 are the thresholds and channels.

Example Datadog monitor config:
```yml
Thresholds:
          Critical: 25.0
          Warning: 100.0
```

This helps improve maintainability, reduce boilerplate, number of "alerts" in the system and reduce load on our respective backends running the alert queries.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82928. Feel free to continue the discussion around this there. Thank you!
